apps/docs/src/content/docs/contribution-guidelines/pull-requests.mdx

```diff
@@ -113,4 +113,11 @@ Below is also a checklist to ensure your PR is ready for review.
       <td>PR description follows template guidelines</td>
     </tr>
   </tbody>
-</table>
\ No newline at end of file
+</table>
+
+
+## What is a CODEOWNER?
+
+In our project, CODEOWNERS are typically subject matter experts for specific areas of the codebase. When you submit a pull request, the relevant CODEOWNERS will be automatically notified and requested to review your changes. You can find out who the CODEOWNER is for a specific file or directory by checking the `.github/CODEOWNERS` file.
+
+For more information, you can check out [GitHub's CODEOWNERS documentation](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners).
```

Comment:
You can actually use [docgen](https://fumadocs.vercel.app/docs/headless/mdx/remark-docgen#file-generator) to render the file directly in here, that way they won't have to click that and go into a different tab.

![image](https://github.com/user-attachments/assets/a6045e9b-2828-47cf-87fd-fa585614c6a5)

Reply:
essentially, adding code like this would into the cg be worthwhile @HasithDeAlwis @MFarabi619:

```
```json doc-gen:file
{
  "file": "../../.github/CODEOWNERS",
    "codeblock": {
    "lang": "ini",
    "meta": "title=\"CODEOWNERS file\""
  }
}
```

Reply:
Ohhh very nice, thank you for this suggestion